webapp/components/wpt-results.html

```diff
@@ -243,7 +243,8 @@
         /* Recomputes the list of displayed directories and test files. */
         const displayedNodeMap = new Map();
         const currentPathParts = this.splitPathIntoLinkedParts(this.path);
-
+        const partCountIfTestFile = this.path == '/' ? 1 : currentPathParts.length + 1;
```

Comment:
=== was in removed code, == in added. I seem to remember an eslint setup, is it running and does it enforce any style? Should it?